### DISCUSSION TITLE
How to use a simple widget?

### DISCUSSION THREAD

**Discussion by Pingear** [flame-engine/flame#1297] [Q&A] [answered]
Hey)
Is there any way to add an ordinary widget to the game? Couldn't find a similar example in the documentation.

**Comment by spydon** [comment, marked as answer]
Hi, 
this is not possible. You can by only add widgets as overlays which means that they live on top of the game.
https://docs.flame-engine.org/1.0.0/game.html?highlight=overlays#flutter-widgets-and-game-instances

**Reply by Pingear** [reply]
This explains why I didn't find it in the documentation))
Strange, it seems to me an obvious idea to add your own widget to the game. I understand that this can have a performance impact and should not be overused. But it would be great to be able to do that.

Do you have any plans to implement this functionality?

**Reply by spydon** [reply]
> Do you have any plans to implement this functionality?

We do not, it is more complex than it sounds like.
